packages/fxa-settings/src/components/CardHeader/index.tsx

```diff
@@ -102,6 +161,38 @@ const CardHeader = (props: CardHeaderProps) => {
     );
   }
 
+  if (isSeparateSubheading(props)) {
+    const { serviceName = MozServices.Default } = props;
+    const isDefaultService = isDefaultServiceName(serviceName);
+    const logo = serviceLogos[serviceName];
+    const logoElem = <span>{logo}</span>;
+
+    const subheadingFtlMsgProps = {
+      id: isDefaultService
+        ? props.subheadingWithDefaultServiceFtlId
+        : props.subheadingWithCustomServiceFtlId,
+      // include `vars={{ serviceName }}` if non-default
+      ...(!isDefaultService && { vars: { serviceName } }),
```

Comment:
Should `vars` be excluded from the FtlMsg if there is a `logo`? Currently we include the logo without the serviceName in the string.